site/content/docs/5.3/utilities/borders.md

```diff
@@ -67,6 +67,10 @@ Here is a list of these extra classes:
 
 Or modify the default `border-color` of a component:
 
+{{< design-callout-alert >}}
+These border color variants, which are just **examples illustrating the use of this color utility**, should not be used because they do not respect the Orange Design System specifications.
```

Comment:
```suggestion
These border color variants are just **examples illustrating the use of this color utility**. They should not be used because they do not respect the Orange Design System specifications.
```

Reply:
Indeed, you proposal is "lighter" in style. If we change this message, then we should do the same for both these messages formulated the same way : 
- Components/List-group.md/#horizontal
- Components/nav-tabs.md/#fill&justify

Reply:
Prefer the Julien proposition since it uses simple sentences.

Reply:
Modified in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2064/commits/4b60154da0b6b709c8dcc2338112e99c0d8381dc